Naga fails to build: "unresolved import `back::continue_forward`"

Naga fails to build as follows (which happens to be the way Firefox uses it):

```console
$ cargo build --target=x86_64-pc-windows-msvc --features="dx12 vulkan"
   Compiling naga v22.0.0 (/home/jimb/rust/wgpu/naga)
   Compiling wgpu-core v22.0.0 (/home/jimb/rust/wgpu/wgpu-core)
   Compiling d3d12 v22.0.0 (/home/jimb/rust/wgpu/d3d12)
error[E0432]: unresolved import `back::continue_forward`
    --> naga/src/back/hlsl/writer.rs:1574:19
     |
1574 |         use back::continue_forward::ExitControlFlow;
     |                   ^^^^^^^^^^^^^^^^ could not find `continue_forward` in `back`

error[E0433]: failed to resolve: could not find `continue_forward` in `back`
   --> naga/src/back/hlsl/writer.rs:107:33
    |
107 |             continue_ctx: back::continue_forward::ContinueCtx::default(),
    |                                 ^^^^^^^^^^^^^^^^ could not find `continue_forward` in `back`

error[E0433]: failed to resolve: could not find `continue_forward` in `back`
   --> naga/src/back/hlsl/mod.rs:330:25
    |
330 |     continue_ctx: back::continue_forward::ContinueCtx,
    |                         ^^^^^^^^^^^^^^^^ could not find `continue_forward` in `back`

Some errors have detailed explanations: E0432, E0433.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `naga` (lib) due to 3 previous errors
$ 
```

--- a/naga/src/back/mod.rs
+++ b/naga/src/back/mod.rs
@@ -19,7 +19,7 @@ pub mod wgsl;
 #[cfg(any(hlsl_out, msl_out, spv_out, glsl_out))]
 pub mod pipeline_constants;
 
-#[cfg(any(feature = "hlsl-out", feature = "glsl-out"))]
+#[cfg(any(hlsl_out, glsl_out))]
 mod continue_forward;
 
 /// Names of vector components.

